tests/test_spherePoint.py

```diff
@@ -212,21 +217,29 @@ def testGetVectorValue(self):
             ((0.0, 90.0), afwGeom.Point3D(0.0, 0.0, 1.0)),
         ]
 
-        for lonLat, vector in pointList:
-            # Convert to Point3D.
-            point = SpherePoint(lonLat[0]*degrees, lonLat[1]*degrees)
+        def checkGetVector(point, lon, lat, vector):
+            """Compare a SpherePoint constructed from lon, lat
+            to a point constructed from the matching vector
+            """
             newVector = point.getVector()
             self.assertIsInstance(newVector, afwGeom.Point3D)
             for oldElement, newElement in zip(vector, newVector):
                 self.assertAlmostEqual(oldElement, newElement)
 
             # Convert back to spherical.
             newLon, newLat = SpherePoint(newVector)
-            self.assertAlmostEqual(newLon.asDegrees(), lonLat[0])
-            self.assertAlmostEqual(newLat.asDegrees(), lonLat[1])
+            self.assertAlmostEqual(newLon.asDegrees(), lon.asDegrees())
+            self.assertAlmostEqual(newLat.asDegrees(), lat.asDegrees())
+
+        for lonLatDeg, vector in pointList:
+            # Convert to Point3D.
```

Comment:
Does this comment belong inside `checkGetVector`?

Reply:
No idea. It's gone now.